pkg/cmd/root/root_test.go

```diff
@@ -26,3 +32,85 @@ func TestRootSetsValidLogLevel(t *testing.T) {
 	ConfigureGlobalLogger("panic")
 	assert.Equal(t, log.PanicLevel, log.DefaultLogger.Level)
 }
+
+func TestCheckDuplicateCmds(t *testing.T) {
```

Comment:
This test is long feels like there is a better way you can handle this. I'd break it up into multiple tests, something like:
- Test plain duplicate commands
- Test nested